lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -59,7 +69,44 @@ export async function updateArtifacts({
         constraint: config.constraints?.kustomize,
       });
     }
-    await exec(`helmfile deps -f ${quote(packageFileName)}`, {
+
+    const cmd: string[] = [];
+    // set cache and config files to a path in privateCacheDir to prevent file and credential leakage
+    const helmConfigParameters = [
+      `--registry-config ${upath.join(privateCacheDir(), 'registry.json')}`,
```

Comment:
can we pass these values via ENV?

Reply:
yes.
we can set bellow environment variables
```
HELM_REGISTRY_CONFIG="/path/to/config.json"
HELM_REPOSITORY_CACHE="/path/to/repository"
HELM_REPOSITORY_CONFIG="/path/to/repositories.yaml"
```